### PR TITLE
Update TR translation #6325

### LINKED ISSUE #6325: [i18n] Translation Turkish (TR) 🇹🇷 needs translators
Dear FreshRSS-Community,

please help us to improve our Turkish translations.

While the core developer and many other contributors develop our [FreshRSS ](https://freshrss.org/) app we cannot support the translations in all 20+ languages. 

## Actual Status
Right now about 90% of our text is translated into Turkish.

## How to support us
How to do it:
1) it is very simple
2) all files are stored in `https://github.com/FreshRSS/FreshRSS/tree/edge/app/i18n/tr` ([Link](https://github.com/FreshRSS/FreshRSS/tree/edge/app/i18n/tr))
3) each untranslated text has an inline comment `//  TODO`
4) When the text is translated, just delete the comment
5) If the English term fits as well, please change the comment from `//  TODO` to `//  IGNORE`
6) open a Pull Request and link to this ticket

## Thanks
Thanks a lot. It will help our very international community of users and developers. Your submitted PRs will be published with our next release version [1.24.0](https://github.com/FreshRSS/FreshRSS/milestones) or later

### PR DESCRIPTION
**Pull request by CilgaIscan**
Closes #6325

Changes proposed in this pull request:
- Update Turkish translation

How to test the feature manually:

Pull request checklist:

- [ ] clear commit messages
- [ ] code manually tested
- [ ] unit tests written (optional if too hard)
- [ ] documentation updated

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/edge/docs/en/developers/04_Pull_requests.md).

### CODE REVIEW

**Comment by CilgaIscan** [comment]
I receive an error but cannot understand what is wrong. Could you please help me? @math-GH

**Comment by math-GH** [comment]
> I receive an error but cannot understand what is wrong. Could you please help me? @math-GH

Fixed :)

**Review by math-GH — APPROVED** [review]
Great work! Thanks a lot!
Harika iş! Çok teşekkürler!

File: app/i18n/tr/admin.php [hunk 1]
```diff
@@ -187,33 +187,33 @@
...
 		'apply' => 'Uygula',
-		'changelog' => 'Changelog',	// TODO
+		'changelog' => 'Değişiklik günlüğü',
 		'check' => 'Güncelleme kontrolü',
-		'copiedFromURL' => 'update.php copied from %s to ./data',	// TODO
+		'copiedFromURL' => 'update.php %s lokasyonundan ./data lokasyonuna kopyalandı',
 		'current_version' => 'Mevcut sürümünüz',
 		'last' => 'Son kontrol',
-		'loading' => 'Updating…',	// TODO
+		'loading' => 'Güncelleniyor...',
```
Comment (Alkarex) [inline app/i18n/tr/admin.php]:
```suggestion
		'loading' => 'Güncelleniyor…',
```

Unicode typography

**Comment by math-GH** [comment]
I resolved the merge conflict

**Pull request merged**